src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Configuration status="warn">
+    <Appenders>
+        <Console name="logToConsole" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d [%t] %-5level %logger{36} – %msg%n"/>
+        </Console>
+        <File name="logToFile" fileName="logs/app.log">
+            <PatternLayout pattern="%d [%t] %p %c %m%n"/>
+        </File>
+    </Appenders>
```

Comment:
This part of configuration file resolves [the problem](https://github.com/mate-academy/jv-logger/pull/1171#pullrequestreview-3351021376).